### ISSUE TITLE
Test on my  own dataset

### ISSUE THREAD

**Issue by xjchao7** [yhw-yhw/MOVER#1]
Thank you for your perfect work.

I would like to ask could I test your method on my own data or internet image?

Thank you very much.

**Comment by yhw-yhw** [comment]
Yes, you can. It's a pure optimization method. And it supports a video, multiple images, or a single image. And we will provide the HPS initialization in this week, stay tune.

**Comment by Xianjin111** [comment]
> Yes, you can. It's a pure optimization method. And it supports a video, multiple images, or a single image. And we will provide the HPS initialization in this week, stay tune.

Thank you so much.

**Comment by YounkHo** [comment]
> Yes, you can. It's a pure optimization method. And it supports a video, multiple images, or a single image. And we will provide the HPS initialization in this week, stay tune.

Will the code of this warehouse be updated in the future? I found that the code for camera pose estimation and ground estimation has not yet been provided.

**Comment by yhw-yhw** [comment]
Yes, will start to organize this two part code in the next two weeks. Thanks a lot for your interest.
